app/scripts/views/settings.js

```diff
@@ -150,9 +150,7 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
     },
 
     beforeDestroy: function () {
-      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
-        $('body').removeClass('settings').show();
-      });
+      $('body').removeClass('settings');
```

Comment:
Without the fade it looks jarring. This looks like a bug in how it fades though, we should try to fix that.